### PR TITLE
 SecondStage object can be attached to Tender and two new fields

### PR DESCRIPTION
**Pull request by ColinMaudry**

### CODE REVIEW

File: release-schema.json [hunk 1]
```diff
@@ -1,29 +1,59 @@
...
+          "title": "Successive reduction",
+          "description": "The procedure will take place in successive stages. In each stage, some participants may be eliminated.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "possiblyNoNegotiation": {
+          "title": "Possibly no negotiation",
+          "description": "The procuring entity reserves the right to award the contract on the basis of the initial bids without any further negotiations.",
```
Comment (jpmckinney) [inline release-schema.json]: Perhaps "buyer or procuring entity" to align with both OCDS and eForms?

File: release-schema.json [hunk 2]
```diff
@@ -1,29 +1,59 @@
...
+        },
+        "successiveReduction": {
+          "title": "Successive reduction",
+          "description": "The procedure will take place in successive stages. In each stage, some participants may be eliminated.",
+          "type": [
+            "boolean",
+            "null"
+          ]
+        },
+        "possiblyNoNegotiation": {
```
Comment (jpmckinney) [inline release-schema.json]: I think the term `noNegotiationNecessary` has the right semantics, as I discuss in https://github.com/eForms/eForms/issues/323#issuecomment-517349757

`possiblyNoNegotiation` makes it sound like some external factors independent of the buyer/procuring entity's prerogative might affect whether there are negotiations.

Comment (ColinMaudry) [reply release-schema.json]: I agree my proposal is not ideal, but I still believe the current label is misleading and will mislead OCDS data consumers. I'll fix it for the sake of consistency with eForms.

Comment (jpmckinney) [reply release-schema.json]: We can think of other options, e.g. `mayNotHaveNegotiation`.

Comment (ColinMaudry) [reply release-schema.json]: OK to implement `mayNotHaveNegotiation`?

Comment (ColinMaudry) [reply release-schema.json]: I can't think of a label with a better balance between consistency (with our `hasProperty` boolean fields) and clarity.

Comment (jpmckinney) [reply release-schema.json]: Sounds good, let's go with that.

File: README.md [hunk 3]
```diff
@@ -22,7 +22,9 @@ In the European Union, this extension's fields correspond to [eForms BG-709 (Sec
         "id": "1",
         "secondStage": {
           "minimumCandidates": 5,
-          "maximumCandidates": 50
+          "maximumCandidates": 50,
+          "successiveReduction": true,
+          "possiblyNoNegotiation": false
```
Comment (jpmckinney) [inline README.md:27]: Need to update example, then can merge.

Comment (jpmckinney) [reply README.md:27]: Fixed in 85b9cc3

**Pull request merged**